eclipse-theia/theia
# Fix #9779: add support for window.menuBarVisibility

Signed-off-by: Esther Perelman <esther.perelman@sap.com>, Chaya Lau <chaya.lau@sap.com>






#### What it does
Fix #9779 add support for window.menuBarVisibility 
The pull-request adds the menu Bar Visibility setting to window setting, this allows compact and hidden layout for menu bar.

#### How to test

click on settings -> preferences ->
tap menu bar visibility in the search
menu bar visibility section should be displayed and drop down should appear with the options : 
classic - menu bar should be displayed on the top - as original.
visible - behaves like classic.
hidden - menu bar does not appear anywhere.
custom - Menu is displayed as a compact button in the sidebar.
 
#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
